### ISSUE TITLE
Macro folder is empty when a macro is deleted

### ISSUE THREAD

**Issue by CapivaraTupiniquim** [earlSt1/vtt-macro-folders#31]
(Google Translator) When I delete a macro the macros folder is empty, I need to close it and open it again for the macros to come back.
![image](https://user-images.githubusercontent.com/67530921/135784524-70a9777f-9df6-4102-bb03-bf83d4f892c1.png)
![image](https://user-images.githubusercontent.com/67530921/135784543-18ac8d3c-125e-413c-89b6-4d1778c44b1d.png)

**Comment by earlSt1** [comment]
Hi, are you on Foundry 0.8.x or 0.7.x? 
Can you also check the F12 console and see if you are getting any red errors?

**Comment by CapivaraTupiniquim** [comment]
Good morning! I using version 0.8.9 and no error appears on the console when I delete a macro.

**Comment by earlSt1** [comment]
Can you try with all other modules disabled and see if it still happens?

**Comment by CapivaraTupiniquim** [comment]
I'm testing without other modules and I'm noticing some things I don't know should be happening:

If only the Macro Folders module is active, when deleting a macro the macro folder is not empty, I can also drag a macro from a compendium to the macro folders. When I activate Macro Folders I can drag macros from a compendium and when I delete the macros folder it is not empty, but if I activate libWrapper it is no longer possible to drag macros, it is only possible by clicking on "import", which makes importing into a folder "Default".

Another thing is that if I try to drag a macro from a compendium to the default folder or any other, this error appears:
![image](https://user-images.githubusercontent.com/67530921/136113855-009fa336-0993-476c-bed6-207a5d046de3.png)

Lastly, when I try to import a complete compendium (with Compendium Folders active), the macros are in the default folder and not in the folder I imported, another issue is that the folder color is always white (#29 ).
![image](https://user-images.githubusercontent.com/67530921/136114320-e057ffe1-fc66-40d8-98ed-014e4f37c7a6.png)

**Comment by CapivaraTupiniquim** [comment]
About the error that the macros disappear, I found the culprit! It's the Boss Bar module.

**Comment by earlSt1** [comment]
Wow thanks for all the debugging youve done. Ill check this out right away.
And really? Boss Bars? Sounds like a strange module to interfere with the
macro directory ill see if i can replicate

**Comment by CapivaraTupiniquim** [comment]
I imagine it is not the real problem, but its dependencies: lib - Color Settings and socketlib. And you're welcome!

**Comment by Weissrolf** [comment]
I also just stumbled over the macro list vanishing when I delete a macro. Using the reload button brings it back. Turns out that indeed "lib - Color Settings" is the module which Macro Folders doesn't play nicely with. Since this was last reported in October, are there any plans to fix this?

**Comment by earlSt1** [comment]
> I also just stumbled over the macro list vanishing when I delete a macro. Using the reload button brings it back. Turns out that indeed "lib - Color Settings" is the module which Macro Folders doesn't play nicely with. Since this was last reported in October, are there any plans to fix this?

Hi, yeah sorry for the wait on this. been quite busy recently. I'll see if i can find some time at the weekend to work on this. Thanks for informing me about the `lib - Color Settings` module though will help a lot in testing.
